tests/test_forms.py

```diff
@@ -0,0 +1,33 @@
+from cms.test_utils.testcases import CMSTestCase
+
+from django import forms
+
+from djangocms_versioning.forms import grouper_form_factory
+from djangocms_versioning.test_utils import factories
+from djangocms_versioning.test_utils.polls.models import PollContent
+
+
+class GrouperFormTestCase(CMSTestCase):
+
+    def tearDown(self):
+        grouper_form_factory.cache_clear()
+
+    def test_factory(self):
+        pv = factories.PollVersionFactory()
+
+        form_class = grouper_form_factory(PollContent)
+        form = form_class()
```

Comment:
I don't think you need to do this. You'll just need to check the form fields against `form.base_fields` instead of `form.fields`